app/services/GovSpendService.scala

```diff
@@ -116,4 +117,21 @@ object GovSpendService {
     "UkContributionToEuBudget" -> 0.70
   )
 
+  val taxYear2019: Map[String, BigDecimal] = Map(
```

Comment:
Best practice is to not use `String`s as keys as they have "infinite" carnality and you are trying to represent a finite problem. Consider refactoring this to use a sum type instead like so
```
sealed trait GoodsAndServices

object GoodsAndServices {
  case object Welfare extends GoodsAndServices
  case object Health extends GoodsAndServices
  ...
  case object UkContributionToEuBudget extends GoodsAndServices
}
```
and usage would be
```
val taxYear2019: Map[GoodsAndServices, BigDecimal] = Map(
  Welfare -> 23.5,
  Health -> 20.2,
  ...
  UkContributionToEuBudget -> 1
)
```